plugins/mimecast/komand_mimecast/actions/find_groups/action.py

```diff
@@ -19,34 +16,16 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        # Import variables from connection
-        url = self.connection.url
-        access_key = self.connection.access_key
-        secret_key = self.connection.secret_key
-        app_id = self.connection.app_id
-        app_key = self.connection.app_key
-
         query = params.get(Input.QUERY)
         source = params.get(Input.SOURCE)
         if query:
             data = {"query": query, "source": source}
         else:
             data = {"source": source}
-
-        # Mimecast request
-        mimecast_request = util.MimecastRequests()
-        response = mimecast_request.mimecast_post(
-            url=url,
-            uri=FindGroups._URI,
-            access_key=access_key,
-            secret_key=secret_key,
-            app_id=app_id,
-            app_key=app_key,
-            data=data,
-        )
+        response = self.connection.client.find_groups(data)
 
         try:
-            output = response["data"][0]["folders"]
+            output = response[DATA_FIELD][0]["folders"]
```

Comment:
constants?

Reply:
I will not move it because this field is specific to this action only and has nothing to do with other actions